wordpress-mobile/WordPress-iOS
# [Zendesk] Update Zendesk to 3.0.3

This PR updates Zendesk to 3.0.3 version, which fixes `Module compiled with Swift 5.1 cannot be imported by the Swift 5.1.2 compiler`

## To test:
- Use Xcode 11.2 or 11.2.1
- Make sure it builds
- Run WP iOS and check everything Zendesk related works as before.

Update release notes:
